base/shell/explorer/trayprop.cpp

```diff
@@ -46,7 +46,7 @@ static void SetBitmap(HWND hwnd, HBITMAP* hbmp, UINT uImageId)
 
 class CTaskBarSettingsPage : public CPropertyPageImpl<CTaskBarSettingsPage>
 {
-private: 
+  private:
```

Comment:
????

Reply:
This change was not my own. It was an appvoyer request,

Reply:
The clang-format thing is still work in progress and sometimes gives strange results (e.g. we don't want to use a 2-space indentation, but it appeared it "suggested" it there...)